ecosystem/python/sdk/aptos_sdk/account_address.py

```diff
@@ -167,3 +210,97 @@ def test_token(self):
             base_address, "bob's collection", "bob's token"
         )
         self.assertEqual(actual, expected)
+
+    def test_to_standard_string(self):
+        # Test special address: 0x0
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000000"
+            ).to_standard_string(),
+            "0x0",
+        )
+
+        # Test special address: 0x1
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000001"
+            ).to_standard_string(),
+            "0x1",
+        )
+
+        # Test special address: 0x4
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000004"
+            ).to_standard_string(),
+            "0x4",
+        )
```

Comment:
nit: These two seems a bit redundant

Reply:
Eh, the more cases the better, it's redundant until it's not, I see 0x1 being somehow different and I wanna test for it specifically.